### ISSUE TITLE
Drawing circuit with initialize instruction using mpl

### ISSUE THREAD

**Issue by obliviateandsurrender** [Qiskit/qiskit-terra#4769]
### Information

- **Qiskit Terra version**: 0.14.2
- **Python version**: 3.8.2
- **Operating system**: Linux Mint 20

### What is the current behavior?
Drawing circuit with _initialize_ instruction with matplotlib (mpl) leads to the following error.

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-48-f16977389543> in <module>
     30 
---> 31 qc.draw(output='mpl')

~/.local/lib/python3.8/site-packages/qiskit/circuit/quantumcircuit.py in draw(self, output, scale, filename, style, interactive, line_length, plot_barriers, reverse_bits, justify, vertical_compression, idle_wires, with_layout, fold, ax, initial_state, cregbundle)
    972             output = None
    973 
--> 974         return circuit_drawer(self, scale=scale,
    975                               filename=filename, style=style,
    976                               output=output,

~/.local/lib/python3.8/site-packages/qiskit/visualization/circuit_visualization.py in circuit_drawer(circuit, scale, filename, style, output, interactive, line_length, plot_barriers, reverse_bits, justify, vertical_compression, idle_wires, with_layout, fold, ax, initial_state, cregbundle)
    313                                       initial_state=initial_state)
    314     elif output == 'mpl':
--> 315         image = _matplotlib_circuit_drawer(circuit, scale=scale,
    316                                            filename=filename, style=style,
    317                                            plot_barriers=plot_barriers,

~/.local/lib/python3.8/site-packages/qiskit/visualization/circuit_visualization.py in _matplotlib_circuit_drawer(circuit, scale, filename, style, plot_barriers, reverse_bits, justify, idle_wires, with_layout, fold, ax)
    663                                        reverse_bits=reverse_bits, layout=layout,
    664                                        fold=fold, ax=ax)
--> 665     return qcd.draw(filename)

~/.local/lib/python3.8/site-packages/qiskit/visualization/matplotlib.py in draw(self, filename, verbose)
    513     def draw(self, filename=None, verbose=False):
    514         self._draw_regs()
--> 515         self._draw_ops(verbose)
    516         _xl = - self._style.margin[0]
    517         _xr = self._cond['xmax'] + self._style.margin[1]

~/.local/lib/python3.8/site-packages/qiskit/visualization/matplotlib.py in _draw_ops(self, verbose)
    856                     vec = '[%s]' % param
    857                     self._custom_multiqubit_gate(q_xy, wide=_iswide,
--> 858                                                  text=op.op.label or "|psi>",
    859                                                  subtext=vec)
    860                 elif op.name == 'unitary':

AttributeError: 'Initialize' object has no attribute 'label'
```

### Steps to reproduce the problem

```
qc = QuantumCircuit(1,1)
qc.initialize([1/np.sqrt(2)]*2,0)
qc.draw(output='mpl')

```
### What is the expected behavior?

It should print it with the label 'psi'.

### Suggested solutions
Since only gate instructions has labels, we should use "_getattr_" with a default None value instead of using "_op.op.label_".

**Comment by metacubestormerxii12** [comment]
I might be able to look into this

**Comment by metacubestormerxii12** [comment]
![image](https://user-images.githubusercontent.com/65141281/88243908-898a8880-cc57-11ea-99c8-9403b9b7a9bc.png)

**Comment by metacubestormerxii12** [comment]
Is this the text that should be shown also in the mpl drawer?

**Comment by enavarro51** [comment]
This was fixed in #4616 and should be in the next release.

**Comment by 1ucian0** [comment]
Closed via #4616
